### ISSUE TITLE
Update docs regarding updated composables

### ISSUE THREAD

**Issue by vernaillen** [vernaillen/wpnuxt-module#69]
An example of how to use the usePosts() composable with typed response:

```
import type { Post } from '#graphql-operations'

const posts: Post[] = await usePosts()
```

You can also take a look at the composables pages of the WPNuxt demo: https://demo.wpnuxt.com/composables

**Comment by yosefeliezrie** [comment]
> An example of how to use the usePosts() composable with typed response:
> 
> 
> 
> ```
> 
> import type { Post } from '#graphql-operations'
> 
> 
> 
> const posts: Post[] = await usePosts()
> 
> ```
> 
> 
> 
> You can also take a look at the composables pages of the WPNuxt demo: https://demo.wpnuxt.com/composables

Are all currently available composables listed here https://demo.wpnuxt.com/composables or are there any missing?

**Comment by vernaillen** [comment]
> Are all currently available composables listed here https://demo.wpnuxt.com/composables or are there any missing?

No, that's not all of them. 

I'm updating to documentation to have a complete list on this page: https://wpnuxt.com/getting-started/composables

**Comment by vernaillen** [comment]
@yosefeliezrie 
I just updated both the documentation and the demo to match the changes in how data is returned by the composables.
This change was implemented in this issue: #73 

The list of composables in the documentation is now complete: https://wpnuxt.com/getting-started/composables

I'm closing this issue
